test/keyboard.html

```diff
@@ -193,6 +193,13 @@
         expect(comboBox.opened).to.equal(false);
       });
 
+      it('should stop propagation of the keyboard enter event', () => {
+        const keydownSpy = sinon.spy();
+        document.addEventListener('keydown', keydownSpy);
+        enter();
+        expect(keydownSpy.called).to.be.false;
```

Comment:
I would prefer to use the Sinon spy assertion helpers. E.g.
```js
expect(keydownSpy).to.not.have.been.called;
```
(I think this should work though I haven't used this exact format)

You can see some examples in: https://github.com/vaadin/vaadin-combo-box/blob/master/test/lazy-loading.html#L77

And docs at: https://www.chaijs.com/plugins/sinon-chai/

Reply:
Though this is not a blocker, and otherwise LGTM.

Reply:
Updated.